packages/web/components/RegisterPlayer.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, ButtonGroup, Flex, Text } from '@metafam/ds';
+import React from 'react';
+
+import { Icon3box } from './icons/Icon3box';
+
+export const RegisterPlayer: React.FC = () => {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+      flex={1}
+    >
+      <Text
+        fontSize="2xl"
+        fontFamily="mono"
+        fontWeight="bold"
+        color="white"
+        m={5}
+      >
+        Register your Player account
+      </Text>
```

Comment:
Instead of hardcoding all these props in every location, should use the "Heading" component from Chakra UI since it will have a lot of these props set. Can also make our own "Heading" component that sets the props how we want.

Also noticed that we are setting a lot of things to white. Instead of doing that, we should just change the ColorMode in Chakra to Dark mode: https://chakra-ui.com/color-mode

That way they will default to lighter colored text

Reply:
the dark mode is a great idea, I'll apply it to my PR with new meta components too!

Reply:
seems like we could just wrap the whole app in `<DarkMode>` component from Chakra

Reply:
or we upgrade to chakra@next https://next.chakra-ui.com/docs/migration#colormode

Reply:
I think the upgrade option looks cleanest @krzysu.

Reply:
@krzysu Are you adding these meta components for buttons and other common ui elements on the design-system? If so I'll wait for that PR to be merged and then incorporate those changes in the web package.

Reply:
I'm "exploring" chakra to see how we could use it for our needs, please take a look at https://github.com/MetaFam/TheGame/pull/55 and let me know if you would use any of that.

Reply:
@krzysu @dan13ram yea, lets use chakra@next

Reply:
@krzysu I think we would be using atleast the MetaButton component in multiple places. I'll look at refactoring my code now to use chakra@next.